metadata/ComputerVision__24100000__metadata.json

```diff
@@ -0,0 +1,26 @@
+{
+  "changeLog": "Release v2024.10.0",
```

Comment:
Do we really want to show the clients that we deliver a 24.10 model and in reality to be a 23.10?

Reply:
How would they know? The changelog is the same, we just updated the version tag attached to the model. We cannot have the same changeLog as 2023.10 since that would be very confusing I guess? Let's talk to RM

Reply:
Discussed here: https://uipath-product.slack.com/archives/C01SHTFNN82/p1726568655988019?thread_ts=1720097964.757159&cid=C01SHTFNN82